web/views.py

```diff
@@ -221,12 +221,10 @@ def post(self, request, *args, **kwargs):
         form = self.get_form()
 
         if form.is_valid():
-            for field in self.form_class().fields:
-                request.session[field] = form.data.get(field, "")
+            for field, value in form.clean().items():
+                request.session[field] = value
 
-            return self.form_valid(form)
-        else:
-            return self.form_invalid(form)
+        return self.form_valid(form)
```

Comment:
just leave a quick comment here about the intent so we're not confused later about why we're returning form_valid(form) regardless of whether the form is actually valid

Reply:
I refactored this to have the "form_invalid()" call back in.  Thank you for calling this out.